engine.py

```diff
@@ -129,6 +132,16 @@ def eventFilter(self, obj, event):
         # info very early on.
         self.tk_3dsmax = self.import_module("tk_3dsmaxplus")
 
+        # The "qss_watcher" setting causes us to monitor the engine's
+        # style.qss file and re-apply it on the fly when it changes
+        # on disk. This is very useful for development work,
+        if self.get_setting("qss_watcher", False):
+            self._qss_watcher = QtCore.QFileSystemWatcher(
```

Comment:
we have discussed adding this to various places but so far haven't done it because it won't work in all cases - it's a dev mechanism and it won't work on shared file systems and not on all linux versions. In previous conversations with @sgsteph we decided to add it to core but only for dev descriptors - does that make sense? IMO this should be a generic feature that we add at the core level, not to individual engines.

I realised that there isn't a ticket for this in our backlogs so i just created one.